extension/src/cli/dvc/constants.ts

```diff
@@ -1,9 +1,5 @@
 import { join } from 'path'
 
-export const MIN_CLI_VERSION = '2.30.0'
```

Comment:
[F] Could not import this into the webview from here because path is not available and it brought the whole thing down